crypto/objects/o_names.c

```diff
@@ -25,17 +25,17 @@
  * We define this wrapper for two reasons. Firstly, later versions of
  * DEC C add linkage information to certain functions, which makes it
  * tricky to use them as values to regular function pointers.
- * Secondly, in the EDK2 build environment, the strcasecmp function is
+ * Secondly, in the EDK2 build environment, the OPENSSL_strcasecmp function is
  * actually an external function with the Microsoft ABI, so we can't
  * transparently assign function pointers to it.
```

Comment:
This does not make any sense anymore as OPENSSL_strcasecmp is our function and not something from libc. So please drop all the obj_strcasecmp garbage.